cmd/gitmono/main.go

```diff
@@ -19,7 +19,7 @@ type Options struct {
 	Project       string `short:"p" default:"." description:"The project directory to operate on"`
 	Verbose       bool   `short:"v" description:"Enable verbose loggging"`
 	DryRun        bool   `long:"dry-run" description:"Do not persist any write action"`
-	CommitScheme  string `long:"commit-scheme" default:"common" description:"The scheme parse commit messages with (common, conventional)"`
+	CommitScheme  string `long:"commit-scheme" default:"conventional" description:"The scheme parse commit messages with (common, conventional)"`
```

Comment:
This is a breaking change, because we switch commit scheme default to conventional.